homeassistant/components/sleepiq/number.py

```diff
@@ -19,15 +19,21 @@ async def async_setup_entry(
 ) -> None:
     """Set up the SleepIQ bed sensors."""
     data: SleepIQData = hass.data[DOMAIN][entry.entry_id]
+
     async_add_entities(
         SleepNumberFirmnessEntity(data.data_coordinator, bed, sleeper)
         for bed in data.client.beds.values()
         for sleeper in bed.sleepers
     )
+    async_add_entities(
+        SleepNumberActuatorEntity(data.data_coordinator, bed, actuator)
+        for bed in data.client.beds.values()
+        for actuator in bed.foundation.actuators
+    )
```

Comment:
Usually we try to only call `async_add_entities` once

Reply:
mypy reports an error when I try to combine that into a single list of entities to add
`Argument 1 to "append" of "list" has incompatible type "SleepNumberActuatorEntity"; expected "SleepNumberFirmnessEntity"`

Should there just be a single entity class with a 'type' variable that defines whether it's an actuator or firmness number? They seemed different enough to have separate classes

Reply:
I found examples of how to use NumberEntityDescription, going to use that.